test/cex/cex_mem.c

```diff
@@ -14,10 +17,11 @@ struct st {
 };
 
 extern struct st *nd_st(void);
+extern void memhavoc(void *ptr, size_t size);
 
 int main(int argc, char**argv) {
-    struct st *p;
-    p = nd_st();
+    struct st *p = malloc(sizeof(struct st));
+    memhavoc(p, sizeof(struct st));
```

Comment:
seems like you are removing a test for `exe-cex` to `cex`. But why are you deleting the old test?

Reply:
good point, lets keep old tests but disable them for now